resources/js/Components/Form/MultiSelect.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div>
+
+    <jet-dropdown width="full" :content-classes="['bg-white', 'max-h-64', 'overflow-y-auto']"
+                  :keep-open="true" @closed="$emit('change', selected)" @opened="$emit('opened')"
+    >
+      <template #trigger>
+
+        <div class="my-2 p-1 flex border border-gray-200 bg-white rounded">
+          <div v-if="Object.entries(selected).length === 0" class="flex-1">
+            <input placeholder="Select an option" disabled
+                   class="bg-transparent p-1 px-2 appearance-none outline-none h-full w-full text-gray-800"
+            >
+          </div>
+          <!-- tags like input field -->
+          <div v-else class="flex flex-auto flex-wrap">
+            <div v-for="(option, index) in selected" :key="index">
+              <div class="flex justify-center items-center m-1 py-1 px-2 space-x-2 rounded-full bg-yellow-300 border border-yellow-500">
+                <img v-if="option instanceof Object && 'icon' in option" class="h-8 w-8 rounded-full object-cover " :src="option.icon" :alt="option.text"/>
+                <div class="text-xs font-normal leading-none max-w-full flex-initial">{{ option instanceof String ? option : option.text }}</div>
+                <svg  xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" stroke="currentColor"
```

Comment:
Is there a way we can download this svg as a static file and serve it oursleves?

Reply:
I mean we have the whole svg here so we could either save it as a `.svg` in `resources/images` and import as an`<img>` tag or even make it a component if its own like `ApplicationLogo.vue` does. 

@alexstojda any opnions on this?